environments/icds-staging/app-processes.yml

```diff
@@ -20,10 +22,12 @@ celery_processes:
     sms_queue:
       pooling: gevent
       concurrency: 5
-    email_queue,repeat_record_queue,sumologic_logs_queue,reminder_case_update_queue,icds_aggregation_queue,analytics_queue:
```

Comment:
Could add this as, to make it more explicit that it's not needed here
```
  None:
    repeat_record_queue:
```

Reply:
Done